core/src/Platforms/net45/NetDesktopPlatformProxy.cs

```diff
@@ -138,14 +141,14 @@ public string GetEnvironmentVariable(string variable)
 
         public string GetProcessorArchitecture()
         {
-            if (IsWindows)
-            {
-                return WindowsNativeMethods.GetProcessorArchitecture();
-            }
-            else
-            {
-                return null;
-            }
+            if (IsWindows)
```

Comment:
nit: collapse to ternary operator here (like GetOperatingSystem() below)

Reply:
done